css/main.css

```diff
@@ -1,3 +1,264 @@
+    }
+    li {
+        text-align: center;
+    }
+}
+
+.job {
+    text-align: center;
+    margin-top: 0;
+}
+
+header {
+    background-color: antiquewhite;
+    align-items: center;
+}
+
+li {
+    margin-left: 15px;
+    list-style-type: none;
+    text-align: left;
+    font-size: 0.8rem;
+}
+
+.fa-user-circle-o {
+    color: darkcyan;
+    padding-top: 16px;
+    text-align: right;
+}
+
+.login {
+    color: rgb(39, 18, 231);
+    text-align: left;
+    font-weight: bold;
+    font-size: 0.8rem;
+}
+
+.intro {
+    background-image: url("https://unenvironment.s3.us-east-2.amazonaws.com/s3fs-public/inline-images/1.jpg");
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-right: -15px;
+    margin-left: -15px;
+}
+
+@media (min-width: 768px) {
+    .intro {
+        min-height:500px ;
+    }
+    .ambition {
+        font-size: small;
+        margin: 4rem 0 0 0;
+    }
+}
+
+.ambition {
+    font-size: small;
+    margin: 4rem 0 0 0;
+}
+
+.succees {
+    font-size: 3rem;
+    font-weight: bolder;
+    margin: 0.5rem 0 0 0;
+}
+
+@media (min-width: 768px) {
+   .succeed {
+    font-size: 7rem;
+    font-weight: bolder;
+    margin: 0.5rem 0 0 0;
+    }
+    button {
+        margin-top:2rem;
+    }
+}
+
+button {
+    background-color: black;
+    color: cornsilk;
+    margin:2rem 0 2rem;
+}
+
+.about {
+    margin: 0;
+    background-color: rgb(235, 231, 217);
+}
+
+.photo {
+        display: block;
+        background-image: url("https://n8g5aw6nho-flywheel.netdna-ssl.com/wp-content/uploads/2019/05/coaching.jpg");
+        background-position: center;
+        background-size: cover;
+        margin: 0;
+        background-color:peachpuff;
+        min-height: 30rem;
+}
+
+.pricing {
+    background-image: url("https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcT7p9klDuc56gZoB3UlRtzGLXemC5yG8hvMUxk6ZoRjmdHlpvND");
+    background-position: center;
+    background-size: cover;
+}
+
+.heading {
+    font-size: 1.5rem;
+    font-weight: bolder;
+}
+
+.pricingbox {
+    border-style: solid;
+    border-width: thick;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+h2 {
+    margin-top: 1rem;
+    text-align: center;
+}
+
+h4 {
+    padding: 0 1rem 2rem 1rem;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+}
+
+@media (min-width: 768px) {
+    h2 {
+        margin-top: 3rem;
+    }
+    
+    h4 {
+        padding: 0 5rem 2rem 5rem;
+    }
+ }
+
+hr {
+    width: 70%;
+}
+
+.line {
+    border:2px solid purple;
+}
+.greenbutton {
+    margin-bottom: 3rem;
+    background-color: forestgreen;
+    color: gainsboro;
+}
+
+.moresrv {
+    margin: 2rem 0 2rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+footer {
+    background-color: skyblue;
+}
+
+.contact {
+    font-size: 0.8rem;
+    margin: 1rem 0 0 1rem;
+}
+
+@media (min-width: 768px) {
```

Comment:
Generally, it's better to group all of the media query css together so that you can tell more easily what changes as the screen resizes. For example,

```
@media (min-width: 768px) {
    .form {
        display: flex;
        flex-direction: column;
        padding-top: 3rem;
        padding-left: 3rem;
     }

    .contact {
        margin: 3rem 0 0 3rem;
    }

    h2 {
        margin-top: 3rem;
    }

    h4 {
        padding: 0 5rem 2rem 5rem;
    }
}
```
Could all go inside the same media query

Reply:
I put all media query CSSs in one block.